lib/utils.pm

```diff
@@ -1156,11 +1154,32 @@ sub reconnect_s390 {
     }
     else {
         my $r = wait_serial($login_ready, 300);
-        if ($r =~ qr/Welcome to SUSE Linux Enterprise 15/) {
-            record_soft_failure('bsc#1040606');
-        }
-        elsif (is_sle) {
-            $r =~ qr/Welcome to SUSE Linux Enterprise Server/ || die "Correct welcome string not found";
+        if (is_sle) {
+            if ($r =~ qr/Welcome to SUSE Linux Enterprise Server/) {
+                return;
+            }
+            else {
+                opensusebasetest::record_info('Welcome message not found', 'Welcome string not found, try ssh to see what is going on...');
+                # Try to login via ssh
+                select_console 'root-console' || die "Correct welcome string not found, ssh login was not possible";
+                my $ret;
+                $ret = script_output 'systemctl status';
+                if ($ret =~ /0 queued/) {
+                    diag "No systemd jobs queued";
+                    return;
+                }
+                else {
```

Comment:
I don't see the need for the else when you already returned in before. I would write the above as

```perl
return diag 'No systemd jobs queued' if $ref =~ /0 queued/;
```

but maybe a record_info is even better? Or a "die" which will also put a text thumbnail on the details view? Or what should we try to continue in the job after we detected no jobs are queued? Like `die 'Welcome message not found but unknown error, no known mitigation'`?